### PR TITLE
[Edit] Adding Codebytes in python built-in-functions [all() , breakpoint() and classmethod)

### LINKED ISSUE #2569: [Edit] Python Add Codebytes
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Please add Codebytes to the listed entries.
https://www.codecademy.com/resources/docs/python/built-in-functions/all
https://www.codecademy.com/resources/docs/python/built-in-functions/breakpoint
https://www.codecademy.com/resources/docs/python/built-in-functions/classmethod

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) when working on the PR for this issue.

Bear in mind the following two principles:
1) There should only be one Codebyte per entry listed as `## Codebyte Example` at the end of the entry.
2) The codebytes should be new examples, not copies of the prior examples.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by dakshdeepHERE**
### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

Fixes #2569

### CODE REVIEW

**Comment by dakshdeepHERE** [comment]
Hello maintainers if you are checking this I had one doubt which was should I mention python version too? In codebyte or python is enough?

File: content/python/concepts/built-in-functions/terms/all/all.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/all/all.md:149]: The codebyte is runnable and displays the output, so we don't need to include an output block.

File: content/python/concepts/built-in-functions/terms/all/all.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/all/all.md:133]:
```suggestion
## Codebyte Example
```

Comment (caupolicandiaz) [reply content/python/concepts/built-in-functions/terms/all/all.md:133]: The issue specified one codebyte per entry. Please pick one below and remove the others.

File: content/python/concepts/built-in-functions/terms/breakpoint/breakpoint.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/breakpoint/breakpoint.md:70]: @SSwiniarski @yangc95 @KTom101 This example throws an error when I run it on the docs compiler (it's a relatively new method, introduced in python 3.7). Should we omit this from the codebyte additions?

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@dakshdeepHERE I've added a few comments. Please update the entry, and tag me when you've made changes. Thanks

File: content/python/concepts/built-in-functions/terms/breakpoint/breakpoint.md
Comment (dakshdeepHERE) [reply content/python/concepts/built-in-functions/terms/breakpoint/breakpoint.md:70]: I would suggest the same, also we need to understand pdb properly to use breakpoint to further debug the code. So after the breakpoint example we also have to give an example where we are using pdb. Like:
```python
import pdb
def calculate_sum(numbers):
    total = 0
    for number in numbers:
        total += number
        pdb.set_trace()  
    return total
```

**Comment by dakshdeepHERE** [comment]
@caupolicandiaz All the requested changes are done. About the breakpoint entry I will follow what other maintainers would suggest.

**Review by caupolicandiaz — APPROVED** [review]
@dakshdeepHERE Unfortunately, I think we have to omit the breakpoint function. I've gone ahead and reverted those changes. This is ready for a second review.

File: content/python/concepts/built-in-functions/terms/all/all.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/all/all.md:137]:
```suggestion
Checking if all elements in a list satisfy a condition:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@dakshdeepHERE I did a review and only had one comment.

**Comment by dakshdeepHERE** [comment]
> @dakshdeepHERE Unfortunately, I think we have to omit the breakpoint function. I've gone ahead and reverted those changes. This is ready for a second review.

It's okay.

**Comment by dakshdeepHERE** [comment]
Done @SSwiniarski

**Review by SSwiniarski — APPROVED** [review]

**Comment by SSwiniarski** [comment]
@dakshdeepHERE congrats on your new edits!

Live links:
- https://www.codecademy.com/resources/docs/python/built-in-functions/all
- https://www.codecademy.com/resources/docs/python/built-in-functions/classmethod

**Comment by dakshdeepHERE** [comment]
Thank you @SSwiniarski ❤️

**Pull request merged**
